go/tools/gopackagesdriver/bazel_json_builder.go

```diff
@@ -35,7 +35,11 @@ func (b *BazelJSONBuilder) fileQuery(filename string) string {
 		fp, _ := filepath.Rel(b.bazel.WorkspaceRoot(), filename)
 		filename = fp
 	}
-	return fmt.Sprintf(`kind("go_library|go_test|go_binary", same_pkg_direct_rdeps("%s"))`, filename)
+	addtlKinds := ""
+	if len(additionalKinds) > 0 {
+		addtlKinds = "|" + strings.Join(additionalKinds, "|")
+	}
+	return fmt.Sprintf(`kind("go_library|go_test|go_binary%s", same_pkg_direct_rdeps("%s"))`, addtlKinds, filename)
```

Comment:
Could you extract the default kinds in a slice and then do a single join to get the `|`-separated list?

Reply:
Sounds good will do shortly.

Reply:
Updated